### PR TITLE
Changelog for the release 1.35.0

### PR DESCRIPTION
**Pull request by lei-tang**
Changelog for the release 1.35.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# Release 1.34.0 20-05-2019
+
+- Update googleapis to official repo (#621)
```
Comment (qiwzhang) [inline CHANGELOG.md]: remove this line,  it is not a feature or bug fixing.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+- Set grpc.max_metadata_size to INT_MAX (#620)
+- Append error message to log_message (#616)
+- Add service_agent and service_config_id to log_entries (#615)
+- Use the service_rollout_id from Check/Report response (#613)
+- Fix grpc percent encode reserved code table (#614)
+- Change default timeout (#611)
```
Comment (qiwzhang) [inline CHANGELOG.md]: Change service control calls default timeout

File: CHANGELOG.md (hunk 1 above)
Comment (lei-tang) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (lei-tang) [reply CHANGELOG.md]: Done.

**Pull request merged**